curriculum/challenges/english/01-responsive-web-design/basic-css-cafe-menu/part-002.md

```diff
@@ -11,10 +11,34 @@ Add a `head` element within the `html` element, so you can add a `title` element
 
 # --hints--
 
-Test 1
+You should have an opening `<head>` tag.
 
 ```js
+assert(code.match(/<head>/i));
+```
+
+You should have a closing `</head>` tag.
+
+```js
+assert(code.match(/<head>/i));
+```
+
+You should have an opening `<title>` tag.
+
+```js
+assert(code.match(/<title>/i));
+```
+
```

Comment:
We should probably check if title is nested in head.

Reply:
Done :)